docs/src/operate/c8y/c8y_service_monitoring.md

```diff
@@ -11,59 +11,62 @@ or on the `child` device can be monitored from the **Cumulocity IoT** by sending
 
 ## Send the health status of a service to health topic.
 
-The table below lists the MQTT topics to which the health status message should be sent, and the
-health status message format for both the `thin-edge` and for the `child` device.
+A health status message can be published for any service on a `status/health` channel. The health message has to contain
+at least the `status` of the service.
 
-|Device-type|Thin-edge-Health-status-mqtt-topic|Health-status-message|
-|------|------------------------|---------------------|
-|Thin-edge-device|`tedge/health/<service-name>`|`{"status":"status of the service","type":"type of service"}`|
-|Child-device|`tedge/health/<child-device-id>/<service-name>`|`{"status":"status of the service","type":"type of service"}`|
+**TODO: Link to the Health Status Message reference!**
 
 :::note
 The `status` here can be `up or down` or any other string. For example, `unknown`.
 :::
 
-For example, to monitor the health status of a `tedge-mapper-c8y` service that is running on a `thin-edge.io` device
-one has to send the below message.
+For example, to update the health status of `device/my-device/service/my-test-service` service, one has to send the
```

Comment:
```suggestion
For example, to update the health status of the `my-test-service` service that is running on `my-device`, one has to send the
```

Reply:
The topic id of the service can be anything, so I didn't want to divide it into device name and service name, but to use a single topic id, as also topic ids that don't fit default scheme can be used.